src/OSPSuite.SimModelNative/src/DESolver.cpp

```diff
@@ -213,7 +213,8 @@ namespace SimModelNative
 				throw ErrorData(ErrorData::ED_ERROR, ERROR_SOURCE,"Cannot allocate memory for solution vector");
 
 			//---- perform initial switch update on <initialvalues>
-			_parentSim->PerformSwitchUpdate(initialvalues, simStartTime);
+			set<int> changedVariablesIndicesUsedInParameters;
+			_parentSim->PerformSwitchUpdate(initialvalues, simStartTime, changedVariablesIndicesUsedInParameters);
```

Comment:
By the way this is the second place where Jacobians must be adjusted. 
Because potentially we have events directly at t=0 which are then fired befor the main ODE loop is started